botfront/imports/api/instances/instances.methods.js

```diff
@@ -341,15 +346,18 @@ if (Meteor.isServer) {
 
                     if (loadModel) {
                         await client.put('/model', { model_file: trainedModelPath });
-                        if (process.env.ORCHESTRATOR === 'gke') {
-                            const { modelsBucket } = Projects.findOne(
-                                { _id: projectId },
-                                { fields: { modelsBucket: 1 } },
-                            ) || {};
-                            if (modelsBucket) {
-                                await uploadFileToGcs(trainedModelPath, modelsBucket);
-                            }
+                    }
+
+                    const trainingWebhook = await getAfterTrainingWebhook();
+                    if (trainingWebhook?.url && trainingWebhook?.method) {
+                        const { namespace } = await Projects.findOne({ _id: projectId }, { fields: { namespace: 1 }})
+                        const body = {
+                            projectId,
+                            namespace,
+                            model: Buffer.from(trainingResponse.data).toString('base64'),
```

Comment:
I'm not sure encoding it to text is needed/good, but I guess that's what's already done in `deploy.model`... Which makes this think, this should go outside of `rasa.train`,  in the same file as `deploy.model`, and share the common parts.